relayer/relays/beefy/config.go

```diff
@@ -12,6 +12,7 @@ type Config struct {
 type SourceConfig struct {
 	Polkadot         config.PolkadotConfig `mapstructure:"polkadot"`
 	FastForwardDepth uint64                `mapstructure:"fast-forward-depth"`
+	UpdatePeriod     uint64                `mapstructure:"update-period"`
```

Comment:
Should add a comment that this quantity is in number of blocks rather than seconds

Reply:
https://github.com/Snowfork/snowbridge/pull/1098/commits/92458230cd25e527f362571ae12ceeff6caba099